docs/es/week03/03-1.md

```diff
@@ -242,22 +241,22 @@ Example: Suppose the input $x$ is one dimensional and has size of 100 and $w$ ha
 | Output size: | $\frac{100 - (5-1)}{1}=96$ | $\frac{100 - (5-1)}{2}=48$ |
 -->
 
-### Variantes para el operador convolucional en DCNNs
+### Variantes para el operador convolucional en DCNN
 
-1. **Zancadas (Striding)**: en lugar de desplazar la ventana una entrada a la vez, se puede realizar con un paso más largo (dos o tres entradas a la vez).
-Ejemplo: Suponiendo que el dato de entrada $x$ es de una dimensión con un tamaño de 100 y $w$ tiene un tamaño de cinco. El tamaño resultante con una zancada de uno o dos está expuesto en la siguiente tabla:
+1. **Zancadas (*Striding*)**: en lugar de desplazar la ventana en $x$ una entrada a la vez, se puede realizar con un paso más grande (por ejemplo, dos o tres entradas a la vez).
+Ejemplo: Supón que la entrada $x$ es unidimensional y tiene un tamaño de 100 y que $w$ tiene un tamaño de cinco. El tamaño de la salida con una zancada (*stride*) de uno o dos se muestra en la siguiente tabla:
 
 
-| Stride       | 1                          | 2                          |
-| ------------ | -------------------------- | -------------------------- |
-| Output size: | $\frac{100 - (5-1)}{1}=96$ | $\frac{100 - (5-1)}{2}=48$ |
+| *Stride*       | 1                          | 2                          |
+| ------------   | -------------------------- | -------------------------- |
+| Tamaño de la salida: | $\frac{100 - (5-1)}{1}=96$ | $\frac{100 - (5-1)}{2}=48$ |
 
 
 <!--
 2. **Padding**: Very often in designing Deep Neural Networks architectures, we want the output of convolution to be of the same size as the input. This can be achieved by padding the input ends with a number of (typically) zero entries, usually on both sides. Padding is done mostly for convenience. It can sometimes impact performance and result in strange border effects, that said, when using a ReLU non-linearity, zero padding is not unreasonable.
 -->
 
-2. **Amortiguación (Padding)**: Frecuentemente en el diseño de arquitecturas de redes neuronales profundas, queremos que el resultado de la convolución sea del mismo tamaño que los datos de entrada. Esto puede ser logrado rellenando los extremos (comúnmente ambos) de los datos de entrada con un número (cero es el más comúnmente utilizado). El relleno de datos es realizado por conveniencia. Esto puede impactar el desempeño de la red resultando en artefactos en las orillas, sin embargo, al utilizar el operador ReLU, rellenar con ceros es razonable.
+2. **Relleno (*Padding*)**: Con frecuencia, al diseñar arquitecturas de redes neuronales profundas, queremos que la salida de la convolución sea del mismo tamaño que los datos de entrada. Esto puede lograrse rellenando los extremos de la entrada con un número de (comúnmente) ceros. Este relleno se realiza principalmente por comodidad. A veces puede etenr un impacto negativo en el desempeño y resultar en extraños efectos en los bordes. Habiendo dicho esto, al utilizar una no linealidad como ReLU, rellenar con ceros es razonable.

```

Comment:
error: "A veces puede **etenr**" to "A veces puede **tener**"

Reply:
Agreed! Sorry for having introduced a typo!

Regarding the suggestion, it is not recommended in Spanish to use the hyphen in that case (see [Fundéu](https://www.fundeu.es/consulta/no-lineal-2071/)).